native/src/main/java/io/ballerina/lib/aws/mpm/AwsMpmClient.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com)
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.lib.aws.mpm;
+
+import io.ballerina.runtime.api.Environment;
+import io.ballerina.runtime.api.Future;
+import io.ballerina.runtime.api.values.BError;
+import io.ballerina.runtime.api.values.BMap;
+import io.ballerina.runtime.api.values.BObject;
+import io.ballerina.runtime.api.values.BString;
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
+import software.amazon.awssdk.auth.credentials.AwsCredentials;
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
+import software.amazon.awssdk.auth.credentials.AwsSessionCredentials;
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.services.marketplacemetering.MarketplaceMeteringClient;
+import software.amazon.awssdk.services.marketplacemetering.model.BatchMeterUsageRequest;
+import software.amazon.awssdk.services.marketplacemetering.model.BatchMeterUsageResponse;
+import software.amazon.awssdk.services.marketplacemetering.model.ResolveCustomerRequest;
+import software.amazon.awssdk.services.marketplacemetering.model.ResolveCustomerResponse;
+
+import java.util.Objects;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public final class AwsMpmClient {
```

Comment:
Same as https://github.com/ballerina-platform/module-ballerinax-aws.marketplace.mpe/pull/2#discussion_r1722770844. Thoughts?
```suggestion
public final class Client {
```

Reply:
Fixed with [aea2f01eb6ce24ba0ccf85cb04b84993b28b9f79](https://github.com/ballerina-platform/module-ballerinax-aws.marketplace.mpm/pull/2/commits/aea2f01eb6ce24ba0ccf85cb04b84993b28b9f79)